cli/generator-op-microservice/generators/microservice/templates/gql.service.ts

```diff
@@ -72,8 +72,8 @@ apollo.applyMiddleware({ app });
 const server = process.env.NODE_ENV !== 'test'
   ? https.createServer(
     {
-      key: fs.readFileSync(`./certs/server.key`),
-      cert: fs.readFileSync(`./certs/server.crt`)
+      key: fs.readFileSync(`${process.env.SSL_KEY}`),
+      cert: fs.readFileSync(`${process.env.SSL_CERT}`)
```

Comment:
Simply passing the variables should also work.

```suggestion
      key: fs.readFileSync(process.env.SSL_KEY),
      cert: fs.readFileSync(process.env.SSL_CERT)
```

Reply:
@deshmukhmayur that approach thows the following error

`Argument of type 'string | undefined' is not assignable to parameter of type 'string` on starting microservices